Robinhood/Robinhood.py

```diff
@@ -610,6 +611,30 @@ def get_url(self, url):
         """
 
         return self.session.get(url, timeout=15).json()
+    
+    def get_popularity(self, stock=''):
+        """Get the number of robinhood users who own the given stock
+            
+            Args:
+                stock (str): stock ticker
+
+            Returns:
+                (int): number of users who own the stock
+        """
+        stock_instrument = self.instruments(stock)[0]["id"]
+        url = "{base}{instrument}/popularity/".format(base=self.endpoints['instruments'],instrument=stock_instrument)
+        return self.session.get(url, timeout=15).json()["num_open_positions"]
+
+    def top_movers(self):
+        """Get a list of the top movers
+            
+            Args: None
+
+            Returns:
+                (List): a list of Ticker strings
+        """
+        instrument_list = self.session.get(self.endpoints['top_movers'], timeout=15).json()["instruments"]
```

Comment:
Is there more data being thrown away here? I'd like to know more about the top movers than just their symbols - the percent change, etc

Reply:
The endpoint just returns a link to each of the top mover's instrument endpoint. You can simply use the ticker to get that same information - the value add of the endpoint is that these tickers are tagged as top movers. It does not have the percent change though. 

Something cool that I just discovered that I am going to change though is that the "top-movers" is just a tag and there are a lot of other tags that can be queried as well. I'm going to add in support for other tags like "etf", "100-most-popular", etc. momentarily.